docs/learn/learn-treasury.md

```diff
@@ -19,10 +19,11 @@ OpenGov is live on Kusama and all treasury proposals need to be submitted throug
 The Treasury is a pot of funds collected through a portion of block production rewards, transaction
 fees, slashing, [staking inefficiencies](learn-staking.md#inflation), etc.
 
-The funds held in the Treasury can be spent by making a spending proposal that, if approved by the
-[Council](learn-governance.md#council), will enter a waiting period before distribution. This
-waiting period is known as the _spend period_, and its duration is subject to
-[governance](learn-governance.md), with the current default set to
+The Treasury funds are held in a [system account](./learn-account-advanced.md#system-accounts) not
+accessible by anyone and only the system internal logic can access it. Funds can be spent by making
+a spending proposal that, if approved by the [Council](learn-governance.md#council), will enter a
+waiting period before distribution. This waiting period is known as the _spend period_, and its
+duration is subject to [governance](learn-governance.md), with the current default set to
```

Comment:
Outdated

Reply:
@DrW3RK can you check this please since you worked on the OpenGov page? We might want to go through all the pages that are outdated and provide a summary table with all the differences between Gov 1 and Open Gov for future reference. This can be part of the wiki reorg.

Reply:
Working with Anaelle from Parity to make the updates https://github.com/w3f/polkadot-wiki/issues/4473